flexget/plugins/input/gazelle.py

```diff
@@ -0,0 +1,498 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+from datetime import datetime
+import logging
+
+from flexget import plugin, db_schema
+from flexget.config_schema import one_or_more
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.manager import Session
+from flexget.plugin import PluginError
+from flexget.utils.database import json_synonym
+from flexget.utils.requests import TokenBucketLimiter
+from flexget.utils.search import normalize_unicode
+from sqlalchemy import Column, Unicode, String, DateTime
```

Comment:
A nitpick: We divide imports into four groups:

1. future
2. builtin
3. flexget
4. 3rd party.